lighthouse/helpers/labwhere.py

```diff
@@ -0,0 +1,57 @@
+"""Connect to labwhere to update or retrieve location information
+
+This file contains the following functions:
+
+  * get_locations_from_labwhere - Make an API call to labwhere
```

Comment:
Not sure these function lists at the top of the files are v useful 
- it's easy to see what functions it contains by scanning down
- can get out of date easily - this one is already missing `set_locations_in_labwhere` :)

Reply:
It's the python documentation convention.
https://realpython.com/documenting-python-code/#package-and-module-docstrings
I think I do agree with you, never a major fan of documentation that reproduces information trially obtained from the code for precisely this reason. Currently trying to be as pythonic as I can though, as I'm familiar with the flipside (Non ruby developers writing Ruby in the style of another language.).